src/constants.js

```diff
@@ -508,5 +508,65 @@ export const styles = {
 
   saveInputsWidth: {
     width: "95%"
+  },
+
+  modelCardContainer: {
+    backgroundColor: "rgb(198, 202, 205)",
+    borderRadius: 5,
+    padding: 20,
+    whiteSpace: "normal",
+    overflowY: "scroll",
```

Comment:
We generally switched to `"auto"` so that Windows machines in particular only show a scrollbar when the content is actually scrollable.

Reply:
Fixed. Thanks!